# 2017-02-06: harry.lin <harry.lin@deltaww.com>
# Change nand flash ecc mode to be BCH8.
--- barebox-2011.06.0/arch/arm/configs/comcerto-2k_rv16x-26x_defconfig.org	2017-01-19 14:50:59.488378079 +0800
+++ barebox-2011.06.0/arch/arm/configs/comcerto-2k_rv16x-26x_defconfig	2017-01-19 14:52:28.958377935 +0800
@@ -315,8 +315,8 @@
 # CONFIG_NAND_ECC_HW is not set
 CONFIG_NAND_ECC_HW_SYNDROME=y
 # CONFIG_NAND_ECC_HW_NONE is not set
-# CONFIG_NAND_COMCERTO_ECC_8_HW_BCH is not set
-CONFIG_NAND_COMCERTO_ECC_24_HW_BCH=y
+CONFIG_NAND_COMCERTO_ECC_8_HW_BCH=y
+# CONFIG_NAND_COMCERTO_ECC_24_HW_BCH is not set
 CONFIG_COMCERTO_NAND_ERASE_FBB=y
 CONFIG_NAND_INFO=y
 CONFIG_NAND_BBT=y
